# [ DATA SCIENCE ] Assignment
## [ HYPOTHESIS_TESTING ]
## S.AKHIL
### Gmail : akhilsangamrjasn@gmail.com

## 1. State the hypotheses statement

Null Hypothesis (H0): The mean weekly operating cost matches the theoretical model: 
μ=1,000+5×600=4,000.

Alternative Hypothesis (𝐻𝑎): The mean weekly operating cost is greater than the theoretical model: 
𝜇>4,000. 
Here, only testing for an increase in cost

## 2. Calculate the Test Statistic

In [1]:
import math
from scipy.stats import norm

In [2]:
# Given data
sample_mean = 3050  # Sample mean weekly cost
theoretical_mean = 1000 + 5 * 600  # Theoretical weekly cost
std_dev = 5 * 25  # Standard deviation
sample_size = 25  # Sample size
alpha = 0.05  # Significance level

In [5]:
# Calculate test statistic
t_statistic = (sample_mean - theoretical_mean) / (std_dev / math.sqrt(sample_size))

In [7]:
# Determine critical value for a one-tailed test
critical_value = norm.ppf(1 - alpha)

In [9]:
# Decision
if t_statistic > critical_value:
    conclusion= "Reject the null hypothesis (H0)."
else:
    conclusion = "Fail to reject the null hypothesis (H0)."

In [17]:
# Output results
print("Sample Mean :", sample_mean)
print("Theoretical Mean :", theoretical_mean)
print("Standard Deviation :", std_dev)
print("Sample Size :", sample_size)
print("Test Statistic :", round(t_statistic, 2))
print("Critical Value (Z-alpha):", round(critical_value, 2))
print("Decision:", conclusion)

Sample Mean : 3050
Theoretical Mean : 4000
Standard Deviation : 125
Sample Size : 25
Test Statistic : -38.0
Critical Value (Z-alpha): 1.64
Decision: Fail to reject the null hypothesis (H0).


## 3. Critical Value (alpha=0.05)

Critical Value (Zalpha): 1.64

## 4. Make a Decision

Compare the computed 
𝑡
t-statistic with the critical value:

𝑡>Critical Valuet: Reject 𝐻0​(evidence supports the claim that costs are higher).
Otherwise,
fail to reject 𝐻0 

##  Formula for calculate the Test_Statistic, Crticial_Value, Decision_make using if statement.

Formula:
1. t_statistic = (sample_mean - theoretical_mean) / (std_dev / math.sqrt(sample_size))
2. critical_value = norm.ppf(1 - alpha)
3. if t_statistic > critical_value:
    decision = "Reject the null hypothesis (H0)."
else:
    decision = "Fail to reject the null hypothesis (H0)."

## Steps to Perform the Hypothesis_Testing Statistic value

Step 1: collecting the Given data(sample_mean, theoretical_mean,std_dev,sample_size,alpha = 0.05).

Step 2: Calculate test statistic(t_statistic), critical_value and Making Decison using the if statement.

Step 3: Print all the values

## Values:

1. Sample Mean : 3050

2. Theoretical Mean : 4000

3. Standard Deviation : 125

4. Sample Size : 25

5. Test Statistic : -38.0

6. Critical Value (Zalpha): 1.64

## 5. Conclusion

By Analyze the data provides strong evidence that weekly operating costs are higher than predicted by the model.

The observed sample mean (3,050) is significantly lower than the theoretical mean (4,000), suggesting the costs may actually be lower than expected.